src/about/public-health.html

```diff
@@ -1,30 +1,70 @@
 
 We expect that all attendees respect those who choose to wear a mask and their boundaries.
 
-DjangoCon US also encourages attendees to take a COVID test prior to arriving at the converence.. We ask that people test again before sprints begin on Thursday. We strongly recommend that people wear a mask and have the latest vaccination boosters. If you are not familiar, we suggest you check out the [CDC's Travel guidelines to stay safe](https://wwwnc.cdc.gov/travel/diseases/covid19).
+DjangoCon US also encourages attendees to take a COVID test prior to arriving at the conference, and to have the latest vaccination boosters. If you are not familiar, we suggest you check out the [CDC's Travel guidelines to stay safe](https://www.cdc.gov/covid/index.html). Rapid COVID tests can be made available upon request, <i>however tests will not be required to attend this year</i>.
 
 The conference will feature select areas in the talk rooms where attendees will be required to wear masks. These areas will be some distance (but not separated by walls) from non-masked areas to allow diffusion.
 
-The conference will provide [N95 masks](https://www.stauffersafety.com/Product/3m-9205-bulk-3m-aura-n95-particulate-respirator-mask-701199) for people to wear. We recommend that all attendees wear a certified KN95 / N95 respirator mask (with no outflow vent) while inside. If you prefer to wear a surgical mask, we recommend an ASTM Level 3 rated mask.
+The conference will make available [N95 masks](https://www.stauffersafety.com/Product/3m-9205-bulk-3m-aura-n95-particulate-respirator-mask-701199) for people to wear, upon request. We recommend that all attendees wear a certified KN95 / N95 respirator mask (with no outflow vent) while inside. If you prefer to wear a surgical mask, we recommend an ASTM Level 3 rated mask.
 
-<!--If you prefer to purchase your own PPE, DjangoCon US is offering up to a $50 reimbursement. This should be for higher filtration PPE you acquire yourself (i.e. reusable respirators, etc). Please [email us](mailto:{{site.contact_us_email}}) before the conference to coordinate.-->
+DjangoCon US offers easy refunds if you need to cancel at any time, due to health status.
 
 DjangoCon US reserves the right to increase the requirements of this policy based on COVID-19 infection, hospitalization or positive test rates as reported by the CDC or other authorities. We may also be required to alter our policy to conform with local, state, or federal mandates.
+{% endmarkdown %}
+      </div>
+
+      <section class="relative flex flex-col self-start p-6 prose bg-white lg:prose-lg lg:pt-10 card-effect-3">
+{% markdown %}
+## TLDR: A short-version of the policy is as follows:
+
+-   Masks are recommended (KN95 / N95 preferred)
+-   Tests are not required this year
+-   Specific seating areas that will require masks
```

Comment:
```suggestion
-   Specific seating areas requiring masks will be designated
```

Small tweak for flow